File: 387. First Unique Character in a String/solution.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        count = [0]*26
        for t in s:
            i = ord(t) - ord('a')
            count[i] += 1
            
        for j, t in enumerate(s):
            i = ord(t) - ord('a')
            if count[i] == 1:
                return j
            
        return -1
            

File: 387. First Unique Character in a String/solution.go
func firstUniqChar(s string) int {
    count := make([]int, 26, 26)
    for i:=0; i<len(s); i++ {
        index := s[i] - 'a'
        count[index]++
    }
    for i:=0;i<len(s); i++ {
        index := s[i] - 'a'
        if count[index] == 1 {
            return i
        }
    }
    return -1
}
